packages/expo-core/android/src/main/java/expo/core/interfaces/Consumer.java

```diff
@@ -0,0 +1,7 @@
+package expo.core.interfaces;
```

Comment:
You can delete `expo-core` now 🙂, it has been renamed to `@unimodules/core`, i. e. `org.unimodules.core`

Reply:
Unresolved 😅